.about.yml

```diff
@@ -18,7 +18,8 @@ licenses:
     url: https://github.com/18F/team_api/blob/master/LICENSE.md
 tasks: https://github.com/18F/team_api/issues
 contact:
-- mailto:michael.bland@gsa.gov
+- url: mailto:michael.bland@gsa.gov
+  text: Michael Bland
```

Comment:
Yes! But in a different commit. Will go add.

On Tue, Dec 1, 2015 at 9:36 AM, Carlo Costino <notifications@github.com>
wrote:

> In .about.yml
> <https://github.com/18F/team_api/pull/47#discussion_r46284756>:
>
> > @@ -18,7 +18,8 @@ licenses:
> >      url: https://github.com/18F/team_api/blob/master/LICENSE.md
> >  tasks: https://github.com/18F/team_api/issues
> >  contact:
> > -- mailto:michael.bland@gsa.gov
> > +- url: mailto:michael.bland@gsa.gov
> > +  text: Michael Bland
>
> You should add yourself, you've contributed to this!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/team_api/pull/47/files#r46284756>.
>